deploy/ansible/roles/nginx/templates/site.conf

```diff
@@ -13,6 +13,15 @@ server {
   include gzip_params;
   server_name {{site_fqdn}};
 
+  # serve static files
```

Comment:
pretty sure these two blocks could go away and we could put `root {{public_path}};` in the main block